client/client_test.go

```diff
@@ -11048,13 +11066,23 @@ func testMultipleCacheExports(t *testing.T, sb integration.Sandbox) {
 	}
 	require.NoError(t, err)
 
-	busybox := llb.Image("busybox:latest")
-	st := llb.Scratch()
+	imgName := integration.UnixOrWindows("busybox:latest", "nanoserver:latest")
+	busybox := llb.Image(imgName)
+	st := integration.UnixOrWindows(llb.Scratch(), llb.Image(imgName))
+
 	run := func(cmd string) {
 		st = busybox.Run(llb.Shlex(cmd), llb.Dir("/wd")).AddMount("/wd", st)
 	}
-	run(`sh -c "echo -n foobar > const"`)
-	run(`sh -c "cat /dev/urandom | head -c 100 | sha256sum > unique"`)
+
+	run(integration.UnixOrWindows(
```

Comment:
nit: same here, pls separate out for readability; but I'm not too pedantic on this one in particular since it's easy to the eyes.